README.md

```diff
@@ -118,6 +118,40 @@ something else:
 (wrap-session your-app {:store (redis-store conn {:prefix "your-app-prefix"})})
 ```
 
+Difference from `clojusc/ring-redis-session`
```

Comment:
This docstring needs to be updated before it's merged ... "Difference from `clojusc/ring-redis-session" doesn't make sense in the context of  `clojusc/ring-redis-session` ;-)

The phrasing needs a little work too, but that's no biggie; I can do that after the PR, too.